osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/threads/RecoverKeyProcess.java

```diff
@@ -115,7 +115,7 @@ public void run() {
 
   private DlmsDevice findDevice() {
     try {
-      return this.domainHelperService.findDlmsDevice(this.deviceIdentification, this.ipAddress);
+      return this.domainHelperService.findDlmsDevice(this.deviceIdentification);
```

Comment:
If `this.ipAddress` is not used here anymore, can it be removed?

Reply:
It is still used in the code (in `checkState` and from the `RecoverKeyProcessInitiator`) though you are right in the sense that the IP address from this field is no longer used to set up the device connection.
I will update the state validation and initiation from the `RecoverKeyProcessInitiator` in order to get rid of the `ipAddress` field in `RecoverKeyProcess`.